modules/GTFE_Card.js

```diff
@@ -72,6 +72,7 @@ class GTFE_Card {
 
   /// ################# MISC #########################
   processContentChoice(payload) {
+    const CARD_METEORIC_SWARM = 7;
```

Comment:
1) this is an unusual place to define this constant
2) why not define all the card constants now, which you're at it?

Reply:
The problem is that we don't know map of ints defined in backend. So this is a workaround to let code know what it is and for coder to explain what 7 means. I'm more than happy to hear suggestions how this problem could be solved because I don't like this place either. To define this map somewhere at the beginning of class? All constants? We don't really need all constants... :(

Reply:
I think if any are defined, they all should be defined. and in a place that makes sense - which class `GTFE_Card` seems as good as place as any - as properties of the class. if you don't want to define all of them can you at least define the two you need (METEORIC and PLANETS) as members of GTFE_Card class?